commands/command_migrate_export.go

```diff
@@ -0,0 +1,203 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/git-lfs/git-lfs/errors"
+	"github.com/git-lfs/git-lfs/filepathfilter"
+	"github.com/git-lfs/git-lfs/git"
+	"github.com/git-lfs/git-lfs/git/githistory"
+	"github.com/git-lfs/git-lfs/git/odb"
+	"github.com/git-lfs/git-lfs/lfs"
+	"github.com/git-lfs/git-lfs/tasklog"
+	"github.com/git-lfs/git-lfs/tools"
+	"github.com/spf13/cobra"
+)
+
+func migrateExportCommand(cmd *cobra.Command, args []string) {
+	l := tasklog.NewLogger(os.Stderr)
+	defer l.Close()
+
+	db, err := getObjectDatabase()
+	if err != nil {
+		ExitWithError(err)
+	}
+	defer db.Close()
+
+	rewriter := getHistoryRewriter(cmd, db, l)
+
+	filter := rewriter.Filter()
+	if len(filter.Include()) <= 0 {
+		ExitWithError(errors.Errorf("fatal: one or more files must be specified with --include"))
+	}
```

Comment:
I like this error message, but I'm wondering what it is users would expect iff they didn't provide `--include` or `--exclude`. My intuition tells me that a user invoking `git lfs migrate export --everything` would want _everything_ to be removed from Git LFS, so perhaps a default of `--include=*` (under the semantics that are implemented below) would make sense.

What do you think?

Reply:
Hmm, I don't think we should use `--everything` for this, as it's already being used to specify that all local references should be migrated.

Maybe we should add a separate flag, `--export-all`?

Reply:
I agree, though I wasn't suggesting that `--everything` be used for that purpose. Rather, that `git lfs migrate export` (with `--everything`) would be a common invocation, and that because it is a common invocation, we should also assume `--include="*"` if no `--include` or `--exclude` flag was given

Reply:
Ah okay! I see what you mean.

I mentioned this below but I'm a bit wary of this. I think it's easy to mistype `git lfs migrate export` when you mean to do something more specific, thus migrating everything in your repository. If your repository has lots of large files, this could also lead to a very large number of downloads.

Saying that, this is the way that `git lfs migrate import` works so it may be fine. If we were to implement it I'd like to do it the way `import` does it, i.e., maintaining a list of file extensions that it's done an operation on and adding attributes for each one.

Reply:
Hmm. Good point, I agree that it could be upsetting for users to download many GiB of LFS object data.

That said, I think that we should leave it as-is, since I don't want to mirror the behavior of `migrate import` here, which I think should have generated an error as you have implemented it here.